README.md

```diff
@@ -211,7 +211,27 @@ In the example below,
 
 ![annotation output](https://raw.githubusercontent.com/ovh/ovh-warp10-datasource/master/dist/assets/screenshot-customAllValue.png)
 
+## Query returning Labels
 
+A variable can contain values for a defined Label. For example, to get all the unique values for the key `hostname`, you can specify a query like this in the templating variable *Query* setting.
+```
+[ $ReadToken '~.*' { } ] FIND
```

Comment:
Good practise:
add `'hostname' '~.*'` as filter in the fetch, this will prevent you to have not existing labels name in the `LMAP`

Reply:
I don't get it. Can you give an example?

Reply:
```suggestion
[ $ReadToken '~.*' { 'hostname' '~.*' } ] FIND
```
will force the query to return series with the hostname label